aws/data_source_aws_batch_compute_environment_test.go

```diff
@@ -0,0 +1,119 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccAWSBatchDataSource_ecsCluster(t *testing.T) {
```

Comment:
Minor naming nitpick: I'd rename this to `TestAccAWSBatchComputeEnvironmentDataSource_basic`